_posts/2015-12-14-simply-secure-rails-api.markdown

```diff
@@ -0,0 +1,723 @@
+---
+layout:       post
+title:        Simply Secure Rails API
+author:       paul_franzen
+summary:      Ever needed a simple way to lock down an API?
+image:        http://res.cloudinary.com/wework/image/upload/s--Al86WVSb--/c_fill,fl_progressive,g_face:center,h_1000,q_jpegmini:1,w_1600/v1427921778/engineering/5-ways-to-create-bulletproof-software-bear.jpg
+categories:   rails, api, security, engineering
```

Comment:
we should probably limit these to shorten the url no? We've been going with just "engineering" now, which is redundant, tbh this category stuff needs to be revisited